src/keystore.h

```diff
@@ -75,13 +75,16 @@ class CKeyStore
     //! Support for Sapling full viewing keys
     virtual bool AddSaplingFullViewingKey(
         const libzcash::SaplingFullViewingKey &fvk,
-        const boost::optional<libzcash::SaplingPaymentAddress> &defaultAddr = boost::none) =0;
```

Comment:
probably bikeshedding by now, but does the removal of boost::optional have something to do with the subject of the pr? Or is it just incidentally here?